composer.json

```diff
@@ -21,9 +21,9 @@
   },
   "require-dev": {
     "doctrine/doctrine-bundle": "^1.6",
-    "phpunit/phpunit": "^4.0 || ^5.0 || ^6.0",
     "symfony/finder": "^2.8 || ^3.0 || ^4.0",
-    "symfony/framework-bundle": "^2.8 || ^3.0 || ^4.0"
+    "symfony/framework-bundle": "^2.8 || ^3.0 || ^4.0",
+    "symfony/phpunit-bridge": "^3.3 || ^4.0"
```

Comment:
`^3.3` is only version constraint that works well for `deps=low`

Reply:
I think we can just update to `^3.4` as 3.3 is not maintained anymore anyway. I'll do that on master directly.

Reply:
Ah sorry, I thought this was regarding SF. Misread that, forget my comment then 😄